app/controllers/shipit/api/rollbacks_controller.rb

```diff
@@ -0,0 +1,24 @@
+
+module Shipit
+  module Api
+    class RollbacksController < BaseController
+      require_permission :deploy, :stack
+
+      params do
+        requires :sha, String, length: { in: 6..40 }
+        accepts :force, Boolean, default: false
```

Comment:
How do you intend to use this on the shipit-next side? I think we could always set to true and always rollback, regardless of whether or not the stack is locked.